src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/TruthSensitivityTranche.java

```diff
@@ -0,0 +1,253 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.text.XReadLines;
+
+import java.io.*;
+import java.util.*;
+
+/*
+ * Represents a truth sensitivity tranche in VQSR.
+ * (Package-private because it's not usable outside.)
+ */
+final class TruthSensitivityTranche extends Tranche {
+    private static final int CURRENT_VERSION = 5;
+
+    private static final String DEFAULT_TRANCHE_NAME = "anonymous";
+    private static final String COMMENT_STRING = "#";
+    private static final String VALUE_SEPARATOR = ",";
+    private static final int EXPECTED_COLUMN_COUNT = 11;
```

Comment:
These should be removed in favor of the protected variables in the Tranche class